backend/src/main/kotlin/app/ehrenamtskarte/backend/regions/database/Setup.kt

```diff
@@ -59,6 +63,11 @@ fun insertOrUpdateRegions() {
                 dbRegion.name = eakRegion[1]
                 dbRegion.prefix = eakRegion[0]
                 dbRegion.website = eakRegion[3]
+                dbRegion
+            }
+            val agency = agencies.find { agency -> agency.officialRegionalKeys.any { it.startsWith(eakRegion[2]) } }
+            if (agency != null) {
+                insertOrUpdateFreinetRegionInformation(agency, dbFreinetRegionInformation, regionEntity)
```

Comment:
🔧 The code in the `if` block can be conditionally applied with `?.let {}`. No function level variable necessary here.